spec/CloudCode.spec.js

```diff
@@ -3157,4 +3157,14 @@ describe('afterLogin hook', () => {
 
     await Parse.Cloud.run('contextTest', {}, { context: { a: 'a' } });
   });
+
+  it('afterFind should have access to context', async () => {
+    Parse.Cloud.afterFind('TestObject', req => {
+      expect(req.context.a).toEqual('a');
```

Comment:
Is the context accessible anywhere after `afterFind`, or is that the last stage for the context object?

Reply:
Not 100% sure but I think that's the last stage. `this.context` will obvs still be available on RestQuery but it's not referenced anywhere else, and after `maybeRunAfterFind`, `RestQuery` returns.

Reply:
Okay, just wanted to make sure whether the test case should also modify the context and test whether the change permeates correctly. But that doesn't apply to the last stage.

Reply:
Do you mean if context is mutated in beforeFind, make sure the mutations show in context on afterFind?

Reply:
I meant that the mutation made in afterFind shows in context at the next stage, but apparently there is no next stage, so we're good here. 🙂